src/ESPEasyEth_ProcessEvent.ino

```diff
@@ -0,0 +1,426 @@
+#include "src/Globals/ESPEasyWiFiEvent.h"
+
+/*
+bool unprocessedWifiEvents() {
```

Comment:
All in comments and copy paste of the same functions in another file.
If you don't use it, please remove it.
I do use the global search a lot and these functions will then also show up in the search results even when in comments.

Reply:
Sorry, i've mistaken added this file, it should be removed